spec/features/trips/trips_destroy_spec.rb

```diff
@@ -6,22 +6,61 @@
   So that nobody can give rides on my behalf
   } do
 
-  let(:trip) { create :trip }
-  let(:admin) { create :user, role: "admin" }
-  let(:user) { create :user, role: "member" }
+  let!(:admin){create(:user, role: "admin")}
+  let!(:rider){create(:rider)}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.